python27-app/module-main/taskqueue.py

```diff
@@ -624,4 +657,6 @@ def post(self):
   ('/python/taskqueue/pull/lease_modification', LeaseModificationHandler),
   ('/python/taskqueue/pull/brief_lease', BriefLeaseHandler),
   ('/python/taskqueue/clean_up', CleanUpTaskEntities),
+  ('/python/taskqueue/empty_callback', EmptyCallback),
+  ('/python/taskqueue/task', TaskHandler),
```

Comment:
Not important really, but do we want these alphabetized? I can see they already weren't that's why I'm asking.

Reply:
I think they are ordered the same as the handlers, which were just added in the order they were written. I wouldn't object if somebody wanted to alphabetize them, though.